app/dashboard.rb

```diff
@@ -114,11 +74,86 @@ def options(input)
     end
```

Comment:
- [ ] But for the data storage implementation, your app is still expected to work like the previous basic UI project. I am unable to exit the app. Kindly implement option 7 which allows users to quit the program peacefully.

Reply:
well resolved! ✔️